atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
I also think setting this via manifest is preferrable. You can also combine the `<dpiAware>` element with the new `<dpiAwareness>` element for Windows 10 build 1607 and higher to enable additional per-monitor DPI features.
https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191.aspx